libfreerdp/crypto/test/TestKnownHosts.c

```diff
@@ -46,8 +46,9 @@ static int prepare(const char* currentFileV2, const char* legacyFileV2, const ch
 
 	for (i=0; i<sizeof(hosts)/sizeof(hosts[0]); i++)
 	{
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl);
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc);
+		if (fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl) != sizeof(char) ||
+			fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc) != sizeof(char))
+			goto finish;
```

Comment:
@nfedera yes you could say they are mixed. However it works correctly. We write 1 element (sizeof(char)) with the size of strlen(hosts[i]) and check if one element was successfully written.
Anyway the sizeof(char) is really confusing and misplaced - I'll update it tomorrow.